types/jest/index.d.ts

```diff
@@ -72,6 +73,48 @@ type ExtractEachCallbackArgs<T extends ReadonlyArray<any>> = {
         : 'fallback'
 ];
 
+type FakeableAPI =
+  | 'Date'
+  | 'hrtime'
+  | 'nextTick'
+  | 'performance'
+  | 'queueMicrotask'
+  | 'requestAnimationFrame'
+  | 'cancelAnimationFrame'
+  | 'requestIdleCallback'
+  | 'cancelIdleCallback'
+  | 'setImmediate'
+  | 'clearImmediate'
+  | 'setInterval'
+  | 'clearInterval'
+  | 'setTimeout'
+  | 'clearTimeout';
+
+interface ModernFakeTimersConfig {
+    /**
+     * If set to `true` all timers will be advanced automatically by 20 milliseconds
+     * every 20 milliseconds. A custom time delta may be provided by passing a number.
+     * The default is `false`.
+     */
+    advanceTimers?: boolean | number;
+    /**
+     * List of names of APIs that should not be faked. The default is `[]`, meaning
+     * all APIs are faked.
+     */
+    doNotFake?: FakeableAPI[];
+    /** Whether fake timers should be enabled for all test files. The default is `false`. */
+    enableGlobally?: boolean;
+    /**
+     * Use the old fake timers implementation instead of one backed by `@sinonjs/fake-timers`.
+     * The default is `false`.
+     */
+    legacyFakeTimers?: boolean;
+    /** Sets current system time to be used by fake timers. The default is `Date.now()`. */
+    now?: number;
```

Comment:
Can be `Date` object here. `fakeTimers` option in `jest.config` cannot take `Date`, because it is not JSON serialisable, but here it works.

```suggestion
    now?: number | Date;
```